contrib/golang/filters/http/source/golang_filter.cc

```diff
@@ -904,6 +904,12 @@ CAPIStatus Filter::getStringValue(int id, GoString* value_str) {
     }
     req_->strValue = state.streamInfo().responseCodeDetails().value();
     break;
+  case EnvoyValue::DownstreamLocalAddress:
+    if (!state.streamInfo().downstreamAddressProvider().localAddress()) {
```

Comment:
seems the address must be there, we could simplify this API.
just like the Lua filter does: https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/http/lua/wrappers.cc#L190

Reply:
done, thanks.